<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGLBegin" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>9</y>
      <width>481</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>87</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="w_label_p">
       <property name="text">
        <string>width p.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="w_label_v">
       <property name="text">
        <string>width v.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="r_label_x">
       <property name="text">
        <string>Rotate x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="r_label_y">
       <property name="text">
        <string>Rotate y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="r_label_z">
       <property name="text">
        <string>Rotate z:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSlider" name="w_slider_p">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="w_slider_v">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="r_slider_x">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="r_slider_y">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="r_slider_z">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="m_label_x">
       <property name="text">
        <string>Move x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_label_y">
       <property name="text">
        <string>Move y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_label_z">
       <property name="text">
        <string>Move z:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QSlider" name="m_slider_x">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_slider_y">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_slider_z">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="s_label_x">
       <property name="text">
        <string>Scale x:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="s_label_y">
       <property name="text">
        <string>Scale y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="s_label_z">
       <property name="text">
        <string>Scale z:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>541</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QSlider" name="s_slider_x">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="s_slider_y">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="s_slider_z">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="file_label">
       <property name="text">
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="count_v_label">
       <property name="text">
        <string>Amount of vertexes:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="count_p_label">
       <property name="text">
        <string>Amount of polygons:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actioncube_obj"/>
    <addaction name="actionfile1_obj"/>
    <addaction name="actionfile2_obj"/>
    <addaction name="actionfile3_obj"/>
   </widget>
   <widget class="QMenu" name="menuType">
    <property name="title">
     <string>Type </string>
    </property>
    <addaction name="actioncentral"/>
    <addaction name="actionparallel"/>
   </widget>
   <widget class="QMenu" name="menuPolygon">
    <property name="title">
     <string>Polygon</string>
    </property>
    <addaction name="actioncolor"/>
    <addaction name="actiondotted"/>
    <addaction name="actionsolid"/>
   </widget>
   <widget class="QMenu" name="menuVertex">
    <property name="title">
     <string>Vertex</string>
    </property>
    <addaction name="actioncolor_2"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="actionnone"/>
    <addaction name="actioncircle"/>
    <addaction name="actionsquare"/>
   </widget>
   <widget class="QMenu" name="menuBackground">
    <property name="title">
     <string>Background</string>
    </property>
    <addaction name="actionred"/>
    <addaction name="actiongreen"/>
    <addaction name="actionblue"/>
    <addaction name="actionpink"/>
    <addaction name="actionyellow"/>
    <addaction name="actionchoose"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionbmp"/>
    <addaction name="actionjpeg"/>
   </widget>
   <widget class="QMenu" name="menuScreencast">
    <property name="title">
     <string>Screencast</string>
    </property>
    <addaction name="actiongo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuType"/>
   <addaction name="menuPolygon"/>
   <addaction name="menuVertex"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuBackground"/>
   <addaction name="menuSave"/>
   <addaction name="menuScreencast"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actioncube_obj">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cube.obj</string>
   </property>
  </action>
  <action name="actionfile1_obj">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>file1.obj</string>
   </property>
  </action>
  <action name="actionfile2_obj">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>file2.obj</string>
   </property>
  </action>
  <action name="actionfile3_obj">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>file3.obj</string>
   </property>
  </action>
  <action name="actioncentral">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>central</string>
   </property>
  </action>
  <action name="actionparallel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>parallel</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </action>
  <action name="actionwidth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>width</string>
   </property>
  </action>
  <action name="actiondotted">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>dotted</string>
   </property>
  </action>
  <action name="actionsolid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>solid</string>
   </property>
  </action>
  <action name="actioncolor_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </action>
  <action name="actionsize">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>size</string>
   </property>
  </action>
  <action name="actionnone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>none</string>
   </property>
  </action>
  <action name="actioncircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>circle</string>
   </property>
  </action>
  <action name="actionsquare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>square</string>
   </property>
  </action>
  <action name="actionred">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>red</string>
   </property>
  </action>
  <action name="actiongreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>green</string>
   </property>
  </action>
  <action name="actionblue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>blue</string>
   </property>
  </action>
  <action name="actionpink">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>pink</string>
   </property>
  </action>
  <action name="actionyellow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>yellow</string>
   </property>
  </action>
  <action name="actionchoose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>choose</string>
   </property>
  </action>
  <action name="actionbmp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>bmp</string>
   </property>
  </action>
  <action name="actionjpeg">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>jpeg</string>
   </property>
  </action>
  <action name="actiongo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>go</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGLBegin</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">gui/qglbegin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>